hack/lib/integration.sh

```diff
@@ -25,7 +25,7 @@ function os::integration::compare() {
         fi
     fi
 
-    os::cmd::expect_success "diff -Naupr --ignore-matching-lines 'startTime' --ignore-matching-lines 'name: \w\{8\}\(-\w\{4\}\)\{3\}-\w\{12\}' --ignore-matching-lines 'sha: \w\{40\}' ${actual} ${expected}"
```

Comment:
Can you do the replacement in this function so it's not duped everywhere? It can still be written to do the substitution before committing the diff to keep the on-disk representation static.